### ISSUE TITLE
[BUG] - Error trying to create instance without backups "Backups option must be either enabled or disabled"

### ISSUE THREAD

**Issue by kmpm** [vultr/ansible-collection-vultr#30]
**Describe the bug**
When trying to create an instance it fails with `Backups option must be either enabled or disabled.` even though `backups: false` in the playbook.
I have tried with setting backups to `false`, `no`, `0` with the same result

**To Reproduce**
1. create playbook `vultr.playbook.yml` as example below
2. make sure ssh key exists and VULTR_API_KEY is defined. 
3. `ansible-playbook -i localhost vultr.playbook.yml`
5. See error
```
...
TASK [create vultr instance] ******************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "fetch_url_info": {"body": "{\"error\":\"Backups option must be either enabled or disabled.\",\"status\":400}", "connection": "close", "content-type": "application/json", "date": "Thu, 27 Oct 2022 19:03:22 GMT", "msg": "HTTP Error 400: Bad Request", "server": "nginx", "status": 400, "strict-transport-security": "max-age=31536000", "transfer-encoding": "chunked", "url": "https://api.vultr.com/v2/instances", "x-content-type-options": "nosniff", "x-frame-options": "DENY", "x-robots-tag": "noindex,noarchive", "x-user": <my user name>"}, "msg": "Failure while calling the Vultr API v2 with POST for \"/instances\"."

...
```

**Expected behavior**
Expect playbook to execute without errors and that a instance is created

**Desktop (please complete the following information where applicable:**
 - OS: Ubuntu 22.04 in WSL2 on Windows 11
 - vultr.cloud version:  1.2.0  
```
ansible --version 
ansible [core 2.13.5]
  config file = None
  configured module search path = ['/home/me/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/me/a5e/ansible-role-nats/venv/lib/python3.8/site-packages/ansible
  ansible collection location = /home/me/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/me/a5e/ansible-role-nats/venv/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
```

**Additional context**
```yaml
---
# vultr.playbook.yml
- hosts: localhost
  connection: local
  gather_facts: false
  tasks:
    - name: Gather Vultr regions information
      vultr.cloud.region_info:
      register: result
   
    - name: Pick region
      ansible.builtin.set_fact:
        region: "{{ result.vultr_region_info | selectattr('country', '==', 'SE') | first }}"

    - name: Gather Vultr plans information
      vultr.cloud.plan_info:
      register: result

    - name: Pick plan
      ansible.builtin.set_fact:
        plan: "{{ result.vultr_plan_info | sort(attribute='monthly_cost') | first }}"

    - name: pickings
      ansible.builtin.debug:
        msg: "{{ region.id }}  {{ plan.id }}"

    - name: create vultr instance
      vultr.cloud.instance:
        label: molecule-test
        hostname: molecule-test
        plan: "{{ plan.id }}"
        ddos_protection: no
        backups: false
        enable_ipv6: no
        ssh_keys:
          - mysshkey
        region: "{{ region.id }}"
        os: Debian 11 x64 (bullseye)        
```

**Comment by kmpm** [comment]
It seems as if the `backups` option should be a string with either `'enabled'` or `'disabled`' and not a boolean according to the API, but perhaps there is something wrong in the call and the value isn't 'translated' into the proper string.

https://www.vultr.com/api/#operation/create-instance

**Comment by resmo** [comment]
Fixed by #32
